adev/src/content/examples/drag-drop/src/root-element/app/app.component.ts

```diff
@@ -23,13 +23,12 @@ import {
 export class CdkDragDropRootElementExample implements AfterViewInit, OnDestroy {
   private _overlay = inject(Overlay);
   private _viewContainerRef = inject(ViewContainerRef);
-
-  @ViewChild(TemplateRef) _dialogTemplate!: TemplateRef<any>;
+  _dialogTemplate = viewChild.required(TemplateRef);
```

Comment:
nit: Could you also make it private, it's unused in the template (and also that was hinted by the leading underscore. 
```suggestion
  private _dialogTemplate = viewChild.required(TemplateRef);
```